### PR TITLE
Add option to analyse only mitochondria

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

I have added a new analysis type (mito) to call and annotate only mitochondrial variants.

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by ramprasadn** [comment]
@Lucpen @jemten Could you guys take a look at this PR?

**Review by fellen31 — COMMENTED** [review]
Could you update the docs as well, to add the new analysis_type? And if there are any changes to the output.

File: subworkflows/local/call_structural_variants.nf [hunk 1]
```diff
@@ -74,39 +79,41 @@ workflow CALL_STRUCTURAL_VARIANTS {
...
+            ch_merged_svs = SVDB_MERGE.out.vcf
+            ch_merged_tbi = TABIX_TABIX.out.tbi
+            ch_versions = ch_versions.mix(TABIX_TABIX.out.versions)
+            ch_versions = ch_versions.mix(SVDB_MERGE.out.versions)
+        }
 
     emit:
-        vcf      = SVDB_MERGE.out.vcf  // channel: [ val(meta), path(vcf)]
-        tbi      = TABIX_TABIX.out.tbi // channel: [ val(meta), path(tbi)]
+        vcf      = ch_merged_svs  // channel: [ val(meta), path(vcf)]
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply subworkflows/local/call_structural_variants.nf:116]: SV in mitochondria will be emitted, but not the nuclear genome if that's what you are asking? 😅

[the remark this answers is not in the document]
Comment (fellen31) [reply subworkflows/local/call_structural_variants.nf:116]: Okay, so the mitochondria SVs are not input into SVDB_MERGE and emitted as ch_merged_svs, that is only the nuclear genome?

[the remark this answers is not in the document]
Comment (ramprasadn) [reply subworkflows/local/call_structural_variants.nf:116]: Ah I see what you mean. That does include nuclear genome and mt genome, but the difference is those calls are in vcf format. `mito` analysis at the moment does not generate SV calls in vcf format. I can update the documentation to make this clear.

[the remark this answers is not in the document]
Comment (fellen31) [reply subworkflows/local/call_structural_variants.nf:116]: I think that would be good, to clarify if the SV calls in VCF format contains MT calls or not, just as you have done for the SNV VCFs 😊

[the remark this answers is not in the document]
Comment (ramprasadn) [reply subworkflows/local/call_structural_variants.nf]: Yupe. That's right!

**Review by fellen31 — APPROVED** [review]
Nice!

**Pull request merged**